diff --git a/PSX.sv b/PSX.sv
index d55e799..94833f2 100644
--- a/PSX.sv
+++ b/PSX.sv
@@ -294,7 +294,7 @@ always @(posedge CLK_50M) begin : cfg_block
 				end
 			5: begin
 					cfg_address <= 7;
-					cfg_data <= pffw2 ? 2147483648 : pdbg2 ? 551954751 : pald2 ? 2201376898 : 2537930535;
+					cfg_data <= pffw2 ? 2147483648 : pdbg2 ? 551954751 : pald2 ? 2387452061 : 2650606425;
 					cfg_write <= 1;
 				end
 			7: begin
@@ -1609,8 +1609,8 @@ logic [11:0] hb_end_lut[8];
 logic [11:0] hb_start, hb_end;
 
 // FIXME: this should be adjusted if hsync changes size to maintain center
-assign hb_start_lut = '{12'd63,  12'd50,  12'd36,  12'd31,  12'd24,  12'd0, 12'd0, 12'd0};
-assign hb_end_lut =   '{12'd767, 12'd613, 12'd441, 12'd383, 12'd305, 12'd0, 12'd0, 12'd0};
+assign hb_start_lut = '{12'd65,  12'd52,  12'd38,  12'd32,  12'd25,  12'd0, 12'd0, 12'd0};
+assign hb_end_lut =   '{12'd765, 12'd612, 12'd438, 12'd382, 12'd305, 12'd0, 12'd0, 12'd0};
 
 always_comb begin
 	hb_start = hb_start_lut[video_hResMode];
