diff --git a/Saturn.sv b/Saturn.sv
index abf78ae..05d4228 100644
--- a/Saturn.sv
+++ b/Saturn.sv
@@ -502,8 +502,8 @@ module emu
 					end
 				5: begin
 						cfg_address <= 7;
-						cfg_data <= !pald2 ? (!dotsel2 ? 32'h96F21F6D : 32'h29E3FEC3) :
-						                     (!dotsel2 ? 32'h8A3D70A4 : 32'h1999999A);
+						cfg_data <= !pald2 ? (!dotsel2 ? 32'h9FED705B : 32'h30B5AA1B) :
+						                     (!dotsel2 ? 32'h8DBCEC85 : 32'h1D53CD88);
 						cfg_write <= 1;
 					end
 				7: begin
