diff --git a/SMS.sv b/SMS.sv
index ecb72f0..b781321 100644
--- a/SMS.sv
+++ b/SMS.sv
@@ -331,8 +331,6 @@ pll pll
 	.refclk(CLK_50M),
 	.rst(0),
 	.outclk_0(clk_sys),
-	.reconfig_to_pll(reconfig_to_pll),
-	.reconfig_from_pll(reconfig_from_pll),
 	.locked(locked)
 );
 
@@ -343,56 +341,6 @@ reg         cfg_write;
 reg   [5:0] cfg_address;
 reg  [31:0] cfg_data;
 
-pll_cfg pll_cfg
-(
-	.mgmt_clk(CLK_50M),
-	.mgmt_reset(0),
-	.mgmt_waitrequest(cfg_waitrequest),
-	.mgmt_read(0),
-	.mgmt_readdata(),
-	.mgmt_write(cfg_write),
-	.mgmt_address(cfg_address),
-	.mgmt_writedata(cfg_data),
-	.reconfig_to_pll(reconfig_to_pll),
-	.reconfig_from_pll(reconfig_from_pll)
-);
-
-always @(posedge CLK_50M) begin
-	reg pald = 0, pald2 = 0;
-	reg [2:0] state = 0;
-	reg pal_r;
-
-	pald <= pal;
-	pald2 <= pald;
-
-	cfg_write <= 0;
-	if(pald2 == pald && pald2 != pal_r) begin
-		state <= 1;
-		pal_r <= pald2;
-	end
-
-	if(!cfg_waitrequest) begin
-		if(state) state<=state+1'd1;
-		case(state)
-			1: begin
-					cfg_address <= 0;
-					cfg_data <= 0;
-					cfg_write <= 1;
-				end
-			5: begin
-					cfg_address <= 7;
-					cfg_data <= pal_r ? 2201376125 : 2537930535;
-					cfg_write <= 1;
-				end
-			7: begin
-					cfg_address <= 2;
-					cfg_data <= 0;
-					cfg_write <= 1;
-				end
-		endcase
-	end
-end
-
 wire reset = RESET | status[0] | buttons[1] | cart_download | bk_loading;
 
 //////////////////   HPS I/O   ///////////////////
